bindings/python/tests/test_fn.py

```diff
@@ -37,6 +40,31 @@ async def test_find_device_files():
     assert devices[0].filename() == dev_name
 
 
+@pytest.mark.asyncio
+async def test_find_device_files_single_pe():
+    dev_name = get_first_device_name("/dev/npu*")
+    pe0 = dev_name + "pe0"
+    pe1 = dev_name + "pe1"
+    config = DeviceConfig.from_str(f"{pe0},{pe1}")
+    fd = os.open(f"/dev/{pe0}", os.O_RDWR)
+    devices = await find_device_files(config)
+    os.close(fd)
+    assert len(devices) == 1
+    assert devices[0].filename() == pe1
```

Comment:
best effort에 의존하지 않고 표현이 가능하면 테스트 의도가 더 명확해질 것 같습니다. 다음과 같은 변경은 어떠세요?

```suggestion
    config = DeviceConfig.from_str(f"{pe0},{pe1}")
    devices = await find_device_files(config)
    assert len(devices) == 2
    fd = os.open(f"/dev/{pe0}", os.O_RDWR)
    # Make sure if another pe is still available
    # Please refer to https://github.com/furiosa-ai/device-api/issues/95.
    devices = await find_device_files(DeviceConfig.from_str(f"{pe1}"))
    os.close(fd)
    assert len(devices) == 1
    assert devices[0].filename() == pe1
```

Reply:
reflect at [c02d050](https://github.com/furiosa-ai/device-api/pull/94/commits/c02d05045aeea91177e853922ef45eb2834cfe5f)